src/thor/timedep_reverse.cc

```diff
@@ -276,7 +276,8 @@ inline bool TimeDepReverse::ExpandReverseInner(GraphReader& graphreader,
   uint32_t idx = edgelabels_rev_.size();
   edgelabels_rev_.emplace_back(pred_idx, meta.edge_id, oppedge, meta.edge, newcost, sortcost, dist,
                                mode_, transition_cost,
-                               (pred.not_thru_pruning() || !meta.edge->not_thru()), restriction_idx);
+                               (pred.not_thru_pruning() || !meta.edge->not_thru()), true,
```

Comment:
don't we want to ignore consecutive closures for single directional search algorithms ? why the logic is different from bidirastar ?

Reply:
Not yet @genadz . Ignore closures does not yet work for unidirectional astar algorithms. The main issue is that we have to selectively ignore closures only if they lead up to the destination edge (but we won't know that while expanding in 1 directions, unlike bidir astar where the expansion starts from both ends).

There's work to be done to make sure that we ignore a closure only if it leads upto the destination.

Reply:
honestly im hoping to eventually just remove the unidirectional astar algorithms and move the time dependent stuff to bidirectional a*.

Reply:
@mandeepsandhu got it! thanks

Reply:
yeah the best we could do here is for forward it would work only at the origin and for reverse it would work only at the destination. i suppose we should implement that just to remove a gotcha from the code?

Reply:
@kevinkreiser do you know why `not_thru` pruning is implemented for time-dependent reverse but not forward? I feel this asymmetry is causing more confusion and I don't want to add the closure pruning without fully understanding why it wasn't done for the time-dep fwd algo. Was it simply an oversight?